### ISSUE TITLE
a python method to convert  a Shape into a point cloud without extra dialog but parameter

### ISSUE THREAD

**Issue by FreeCAD-Bug-Importer** [FreeCAD/FreeCAD#5785]
Issue imported from https://tracker.freecad.org/view.php?id=3446

* **Reporter:** microelly2
* **Date submitted:** 4/25/2018
* **FreeCAD version:** 0.18
* **Category:** Feature
* **Status:** new
* **Tags:** Mesh

# Original report text

see 
https://forum.freecadweb.org/viewtopic.php?f=22&t=28366#p229963

# Other bug information

* **Priority:** normal
* **Severity:** minor
* **Category:** Feature
* **Updated:** 2/6/2021

**Comment by Nicolaus93** [comment]
Any development on this?

**Comment by luzpaz** [comment]
@Nicolaus93 we're currently in a feature freeze but you can bump the forum thread and see who weighs-in...?

**Comment by Nicolaus93** [comment]
[This conversation](https://forum.freecadweb.org/viewtopic.php?p=229986&sid=9d09be58fdfbf90981b85440b3e0823e#p229986) is from almost 4 years ago. Should I post another reply on the same thread or do something else? 
![image](https://user-images.githubusercontent.com/13076944/162740795-b5055db4-ea17-4b63-b26a-fb9de68bdc60.png)

**Comment by luzpaz** [comment]
Yes, please  post to the thread and x-post back to this ticket (Since it's migrated to GH from MantisBugtracker)
